PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.hdinsight/src/com/microsoft/azuretools/hdinsight/projects/HDInsightScalaPageTwo.java

```diff
@@ -55,6 +58,40 @@ public void configureJavaProject(String newProjectCompliance, IProgressMonitor m
 			if (parent == null) { 
 				parent = (HDInsightsScalaProjectWizard) this.getWizard();
 			}
+			
+			if (hasConfiguredScalaClasspathContainer == false) {
+				hasConfiguredScalaClasspathContainer = true;
+				Display.getDefault().syncExec(new Runnable() {
```

Comment:
Lambda expression `() -> {}` would be simple to replace `new Runnable() { public void run() {`:

Reply:
Thanks! Fixed

---
In reply to: [144226014](https://github.com/Microsoft/azure-tools-for-java/pull/1098#discussion_r144226014) [](ancestors = 144226014)